pkg/cmd/experimental/diagnostics/master.go

```diff
@@ -0,0 +1,143 @@
+package diagnostics
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	diagnosticflags "github.com/openshift/origin/pkg/cmd/experimental/diagnostics/options"
+	"github.com/openshift/origin/pkg/diagnostics/log"
+	masterdiagnostics "github.com/openshift/origin/pkg/diagnostics/master"
+	systemddiagnostics "github.com/openshift/origin/pkg/diagnostics/systemd"
+	diagnostictypes "github.com/openshift/origin/pkg/diagnostics/types/diagnostic"
+)
+
+const (
+	MasterDiagnosticsRecommendedName = "master"
+
+	StandardMasterConfigPath string = "/etc/openshift/master/master-config.yaml"
+)
+
+var (
+	AvailableMasterDiagnostics = util.NewStringSet("AnalyzeLogs", "UnitStatus", "MasterConfigCheck")
```

Comment:
Diagnostics need to be initialized and no two diagnostics look the same.  That means you can't initialize them into a static map as you did before.  This list is local to where they are used.  Constants sound reasonable.